includes/api/create-cart.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * create cart 
+ */
+function createCartData(WP_REST_Request $request)
+{
+    rzpLogInfo("fetchCartData");
+    global $woocommerce;
+    $params           = $request->get_params();
+    $logObj           = array();
+    $logObj['api']    = 'createCartData';
+    $logObj['params'] = $params;
+
+    $couponCode = null;
+
+    intiCartCommon();
```

Comment:
if this is not being used in case of pdp checkout, can it be put below the if block?